v22.2/cloud-storage-authentication.md

```diff
@@ -49,7 +49,7 @@ Assume role authentication allows you to use temporary short-lived credentials t
 ## Amazon S3 implicit
 
 {{site.data.alerts.callout_info}}
-`implicit` authentication **cannot** be used to run bulk operations from {{ site.data.products.db }} clusters—instead, use `AUTH=specified`.
+To use `implicit` authentication on a {{ site.data.products.db }} cluster, it is necessary to authenticate using assume role authentication. See [GAmazon S3 assume role](#amazon-s3-assume-role) for more details.
```

Comment:
Is the "GAmazon" here a typo? or is it needed for some magic?

Reply:
That is definitely a typo, thanks for catching.